### ISSUE TITLE
Is it works for amd64 architecture?

### ISSUE THREAD

**Issue by vldmr-k** [odoo-it/docker-odoo#39]
```
 => [odoo internal] load build definition from Dockerfile                                                                                                                                                                        0.0s
 => => transferring dockerfile: 451B                                                                                                                                                                                             0.0s
 => ERROR [odoo internal] load metadata for ghcr.io/odoo-it/docker-odoo:17.0-latest                                                                                                                                              0.3s
------
 > [odoo internal] load metadata for ghcr.io/odoo-it/docker-odoo:17.0-latest:
------
failed to solve: ghcr.io/odoo-it/docker-odoo:17.0-latest: no match for platform in manifest: not found

```

**Comment by ivantodorovich** [comment]
Hello @vldmr-k,

Thanks for reaching out.

Indeed, there seems to be an issue with how we're building the multi architecture.
The `linux/amd64` and `linux/arm64` images are built in parallel jobs, and separately pushed to the registry.
It seems that, if we want to do that, we need to manually merge and push the manifest list.

I'm currently working on a fix

**Comment by ivantodorovich** [comment]
This issue should be solved with version [3.0.1](https://github.com/odoo-it/docker-odoo/releases/tag/3.0.1).
Let me know otherwise :) 

Thanks!
